generated/models/SwapResponseDynamicSlippageReport.ts

```diff
@@ -43,8 +43,33 @@ export interface SwapResponseDynamicSlippageReport {
      * @memberof SwapResponseDynamicSlippageReport
      */
     slippageBps?: number;
+    /**
+     * 
+     * @type {string}
+     * @memberof SwapResponseDynamicSlippageReport
+     */
+    categoryName?: SwapResponseDynamicSlippageReportCategoryNameEnum;
```

Comment:
this can just do 
`categoryName?: 'stable' | 'lst' | 'bluechip' | 'verified'`

Reply:
everything in `generated/` folders can ignore